samples/sensor/bma400/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020 Bosch Sensortec GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c0 {
```

Comment:
You can make it much more generic. As there is not any board in the tree equipped this device, you can rename this overlay to app.overlay, move it to  _samples/sensor/bma400/_ , and rename i2c0 to arduino_i2c. With this changes one can use this sample on any board which provides a configuration for Arduino connectors.

```diff
diff --git a/samples/sensor/bma400/boards/nrf52840dk_nrf52840.overlay b/samples/sensor/bma400/boards/nrf52840dk_nrf52840.overlay
index 10bf917c5a..b28abb5995 100644
--- a/samples/sensor/bma400/boards/nrf52840dk_nrf52840.overlay
+++ b/samples/sensor/bma400/boards/nrf52840dk_nrf52840.overlay
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-&i2c0 {
+&arduino_i2c {
        bma400@14 {
                compatible = "bosch,bma400";
                reg = <0x14>;
```